gdk4/src/rectangle.rs

```diff
@@ -59,3 +59,21 @@ impl From<RectangleInt> for Rectangle {
         unsafe { *(&r as *const _ as *const _) }
     }
 }
+
+
+impl From<cairo::Rectangle> for Rectangle {
```

Comment:
This might be a bad idea actually... because converting an f64 to i32 might fail, we should probably at least use try_into and panic with a proper error message

Reply:
Or even implement TryFrom

Reply:
Actually, never mind